api/core/domain/models/model_datas_mapping.py

```diff
@@ -1261,6 +1262,126 @@ def _build_model_datas():
             model=Model.DEEPSEEK_V3_0324,
             display_name="DeepSeek V3 (latest)",
         ),
+        Model.GROK_3_BETA: ModelData(
+            display_name="Grok 3 (beta)",
+            supports_json_mode=True,
+            supports_input_image=True,
```

Comment:
<img width="394" alt="Screenshot 2025-04-11 at 13 08 51" src="https://github.com/user-attachments/assets/f354e7cf-10c8-4509-9843-fbc6681abd63" />
@guillaq I think Grok 3s do not support images, right ?